### DISCUSSION TITLE
Error "No type marked with `[SeqApp]` could be found." trying to install from local feed

### DISCUSSION THREAD

**Discussion by 0xVoyager2** [datalust/seq-tickets#1898] [Help]
Hi, I've been dabbing in this app from 6 years ago to let me post events to a Discord Webhook (https://github.com/glkn/Seq.App.Discord). Since it has been 6 years since last updated, discord changed some of it's api and how things work. So I forked the project and tried my best to make it work to the current API and discord's standards.

I updated the dependencies, the .net target framework (was on 4.5, updated to 4.8), packaged it with Nuget CLI and upload it to a server.
When trying to install it on a Seq instance running on a Docker Container it gives me the following error:

"Package Seq.App.Discord could not be interrogated for app configuration: No type marked with `[SeqApp]` could be found."

What can I do to fix this?
![imagem_2023-07-22_044236301](https://github.com/datalust/seq-tickets/assets/4868121/b5a92e9d-7293-45b6-a3cb-4cef5686905c)

Thanks in advance!

[Edit 1] :Also, I've updated both Seq.Apps dependency and the running instance.

**Comment by liammclennan** [comment]
Hi @andrejmonteiro ,

The instructions for creating a Seq app are at https://docs.datalust.co/docs/writing-seq-apps.

**Reply by 0xVoyager2** [reply]
It's not like I'm creating an app from scratch, I'm forking someone else's repo to fix something unrelated to the seq-apps but got this error while doing so.

**Reply by nblumhardt** [reply]
Hi André,

The Seq app subsystem has also changed a bit in the intervening six years; I think it should all come together pretty easily, but your best bet is to clone:

https://github.com/datalust/seq-app-httprequest

and grab the CSPROJ file, Build.ps1, and project layout, so that you can skip the process of incrementally discovering all the new differences.

The CSPROJ and build script include things like the switch to .NET (core) as the target platform, the method for including/excluding dependencies, etc. - you should be able to just rename the CSPROJ, delete the .cs files, and replace them with yours, without needing to do too much else.

Let me know if this helps! If not, it would be good to know the Seq version you're targeting/testing against, too.

Nick
